components/ai_chat/resources/page/components/alerts/long_conversation_info.tsx

```diff
@@ -14,8 +14,7 @@ import getPageHandlerInstance from '../../api/page_handler'
 export default function LongConversationInfo() {
   const context = React.useContext(DataContext)
 
-  const handleClearChat = (e: any) => {
-    e.preventDefault()
```

Comment:
Can you confirm this won't break something @nullhook?

Reply:
just tested and it doesn't break